package-lock.json

```diff
@@ -1931,7 +1931,6 @@
 				"jest-resolve": "^26.6.2",
 				"jest-util": "^26.6.2",
 				"jest-worker": "^26.6.2",
-				"node-notifier": "^8.0.0",
```

Comment:
I'll bump these separately after this PR too; let's remove these for now.

\+ the puml diffs from #1130 below